.github/ISSUE_TEMPLATE/feature.yml

```
@@ -7,9 +7,11 @@ body:
     attributes:
       value: |
         Thank you for taking time to suggest your idea! Please be aware that this process takes a while (at least weeks). We appreciate your patience!
-        
+
         If you haven't already, [please go search the existing issues for ideas similar to yours](https://github.com/django/new-features/issues?q=is%3Aissue).
-        
+
+        Where possible, please don't make multiple feature suggestions in a single issue. Open multiple issues for distinct suggestions.
```

```suggestion
        Please use a single issue for a one feature suggestions. If you have related ideas, please create multiple issues which reference each other.
```
I feel somewhat strongly about the first sentence which is bit clearer on what a person should do rather than what they shouldn't. I'm not sure if the second is needed. I wanted to mention "write a blog post to advocate for your whole vision", but that's feels even more out of scope for this.

---

In hindsight, I agree. It's much nicer to tell someone what _to_ do, rather than what _not to_ do.